fooyin fails to compile on Fedora 44 (Rawhide)

**Describe the bug**
fooyin compilation fails with an error when building it on Fedora 44 (Rawhide). The error message is:
```
...
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:126:17: error: ‘QElapsedTimer’ was not declared in this scope; did you mean ‘QBasicTimer’?
  126 |     std::vector<QElapsedTimer> m_lastPeakTimers;
      |                 ^~~~~~~~~~~~~
      |                 QBasicTimer
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:126:30: error: template argument 1 is invalid
  126 |     std::vector<QElapsedTimer> m_lastPeakTimers;
      |                              ^
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:126:30: error: template argument 2 is invalid
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:126:10: error: ‘<expression error>’ in namespace ‘std’ does not name a type
  126 |     std::vector<QElapsedTimer> m_lastPeakTimers;
      |          ^~~~~~~~~~~~~~~~~~~~~
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:149:5: error: ‘QElapsedTimer’ does not name a type; did you mean ‘QBasicTimer’?
  149 |     QElapsedTimer m_elapsedTimer;
      |     ^~~~~~~~~~~~~
      |     QBasicTimer
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp: In member function ‘void Fooyin::VuMeter::VuMeterWidgetPrivate::reset()’:
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:185:27: error: ‘m_lastPeakTimers’ was not declared in this scope
  185 |     std::ranges::for_each(m_lastPeakTimers, &QElapsedTimer::start);
      |                           ^~~~~~~~~~~~~~~~
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:185:46: error: ‘QElapsedTimer’ has not been declared; did you mean ‘QBasicTimer’?
  185 |     std::ranges::for_each(m_lastPeakTimers, &QElapsedTimer::start);
      |                                              ^~~~~~~~~~~~~
      |                                              QBasicTimer
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp: In member function ‘void Fooyin::VuMeter::VuMeterWidgetPrivate::calculatePeak()’:
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:226:32: error: ‘m_elapsedTimer’ was not declared in this scope; did you mean ‘elapsedTime’?
  226 |     const qint64 elapsedTime = m_elapsedTimer.restart();
      |                                ^~~~~~~~~~~~~~
      |                                elapsedTime
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp: In member function ‘void Fooyin::VuMeter::VuMeterWidgetPrivate::updateChannelLevels(int, qint64, qint64, float, bool&)’:
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:260:23: error: ‘m_lastPeakTimers’ was not declared in this scope
  260 |     auto& peakTimer = m_lastPeakTimers.at(channel);
      |                       ^~~~~~~~~~~~~~~~
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp: In member function ‘void Fooyin::VuMeter::VuMeterWidgetPrivate::playStateChanged(Fooyin::Player::PlayState)’:
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:586:13: error: ‘m_elapsedTimer’ was not declared in this scope
  586 |             m_elapsedTimer.start();
      |             ^~~~~~~~~~~~~~
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp: In member function ‘void Fooyin::VuMeter::VuMeterWidget::renderBuffer(const Fooyin::AudioBuffer&)’:
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:661:8: error: ‘class Fooyin::VuMeter::VuMeterWidgetPrivate’ has no member named ‘m_lastPeakTimers’
  661 |     p->m_lastPeakTimers.resize(channels);
      |        ^~~~~~~~~~~~~~~~
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp: In lambda function:
/builddir/build/BUILD/fooyin-0.9.2-build/fooyin-0.9.2/src/plugins/vumeter/vumeterwidget.cpp:709:20: error: ‘class Fooyin::VuMeter::VuMeterWidgetPrivate’ has no member named ‘m_lastPeakTimers’
  709 |                 p->m_lastPeakTimers.at(i).start();
      |                    ^~~~~~~~~~~~~~~~
...
```

**To Reproduce**
1. Try building fooyin 0.9.2 on Fedora Rawhide.

**Expected behaviour**
Successful build.

**Screenshots**
N/A

**System:**
 - OS: Fedora Rawhide (44)
 - fooyin version: 0.9.2

**Additional context**
Build started failing after Qt6 was updated to 6.10rc in Fedora: https://koschei.fedoraproject.org/build/21492163 .

It looks like `#include <QElapsedTimer>` is missing from `src/plugins/vumeter/vumeterwidget.cpp`. The use of `QElapsedTimer` class requires that include, per [Qt6 documentation](https://doc.qt.io/qt-6/qelapsedtimer.html). I'll post a PR adding it shortly.

--- a/src/plugins/vumeter/vumeterwidget.cpp
+++ b/src/plugins/vumeter/vumeterwidget.cpp
@@ -32,8 +32,8 @@
 
 #include <QActionGroup>
 #include <QBasicTimer>
-#include <QElapsedTimer>
 #include <QContextMenuEvent>
+#include <QElapsedTimer>
 #include <QJsonObject>
 #include <QMenu>
 #include <QPainter>

--- a/src/plugins/vumeter/vumeterwidget.cpp
+++ b/src/plugins/vumeter/vumeterwidget.cpp
@@ -32,6 +32,7 @@
 
 #include <QActionGroup>
 #include <QBasicTimer>
+#include <QElapsedTimer>
 #include <QContextMenuEvent>
 #include <QJsonObject>
 #include <QMenu>

